libs/server/repository/api-rest/src/lib/controllers/repository.controller.ts

```diff
@@ -1,16 +1,22 @@
-import { Body, Controller, Post } from '@nestjs/common';
+import { Body, Controller, Get, Post } from '@nestjs/common';
 import { ApiTags } from '@nestjs/swagger';
 import {
   AddRepositoryCommand,
   RepositoryFacade
 } from '@pimp-my-pr/server/repository/core/application-services';
 import { AddRepositoryDto } from '../dtos/add-repository.dto';
+import { RepositoryEntity } from '@pimp-my-pr/server/repository/core/domain';
 
 @ApiTags('repository')
 @Controller()
 export class RepositoryController {
   constructor(private repositoryFacade: RepositoryFacade) {}
 
+  @Get('repositories')
```

Comment:
maybe add global prefix 'repository' in the Controller decorator, and remove 'repositories' from this Get and from Post above addRepository :)